dev/AppAttach/AppAttachFramework/AppAttachAPI/Exceptions/Messages.cs

```diff
@@ -0,0 +1,71 @@
+﻿// Copyright (c) Microsoft Corporation and Contributors.
+// Licensed under the MIT License.
+
+using System.Collections;
+using System.Reflection;
+using System.Xml;
+
+namespace AppAttachAPI.Exceptions
+{
+    /// <summary>
+    /// A static class containing error messages for various publishing-related errors.
+    /// </summary>
+    public static class Messages
+    {
+        private static Hashtable _errorTable;
+
+        // TODO: Add support for localization
```

Comment:
TODOs should reference ADO work items e.g.

```
/// TODO:12345 Add support for localization
```

where `12345` corresponds to the ADO item e.g. `https://task.ms/12345` would resolve to it (if whoever clicked on such a link has access)